tools/proto2jsonschema/README.md

```diff
@@ -0,0 +1,30 @@
+# proto2jsonschema
+
+This tool converts proto event definitions to JSON Schemas.
+
+## Requirements
+
+This tool requires the following languages:
+
+- `go version`: `1.13+`
+- `node -v`: `v10+`
+
+### Run
+
+From the root directory of this repo, run this script:
+
+```sh
+./tools/proto2jsonschema/gen.sh
+```
+
+This script:
+- Installs protobuf tools: `protoc`
+- Installs protobuf extension for JSON Schemas: `chrusty/protoc-gen-jsonschema`
+- Clones `googlapis/googleapis`
```

Comment:
Why would we clone this when a) you've introduced a submodule for it and b) all the required protos are expected to be in `third_party/googleapis`?

Reply:
There should be no cloning of googleapis, this was never used as part of the `protoc` generation process and was a mistake adding it. This instruction is outdated.